src/helpers/turbo.ts

```diff
@@ -0,0 +1,90 @@
+import snapshot from '@snapshot-labs/snapshot.js';
+import fetch from 'node-fetch';
+import db from './mysql';
+
+const SCHNAPS_API_URL = process.env.SCHNAPS_API_URL || 'https://schnaps.snapshot.box';
+const NO_SCHNAPS = process.env.NO_SCHNAPS === 'true';
```

Comment:
Could we default this to true if `process.env.SCHNAPS_API_URL` is empty? This will make both these env optional. good for new devs, else not important

Reply:
good idea.
removed the default `SCHNAPS_API_URL`, removed `NO_SCHNAPS`, simplified the code: c1136b9e618f6d8b5f1e462a09a2cf7ca50dbad0